### PR TITLE
Couple changes for css test

### PR DESCRIPTION
**Pull request by catesandrew**
These are some minor changes to support creating a html output page from css test.

### CODE REVIEW

File: test/run-css.js [hunk 1]
```diff
@@ -23,15 +24,20 @@ var config = test.config(),
...
+    userAgent: userAgent,
+    loadPlugins: false
   },
   viewportSize: {width: width, height: height}
 });
 
 css.init({
   screenshotRoot: './test/screenshots/' + dirname,
-  failedComparisonsRoot: './build/css-failures/' + dirname,
+  //failedComparisonsRoot: './build/css-failures/' + dirname,
```
Comment (kpdecker) [inline test/run-css.js:35]: Why is this being removed?

File: test/run-css.js [hunk 2]
```diff
@@ -45,7 +51,15 @@ css.init({
...
 casper.start();
 
-test.test(casper, css, platform, port);
+test.test({
+  casper: casper,
+  css: css,
+  platform:platform,
+  port: port,
+  dirname: dirname,
+  child_process: child_process,
```
Comment (kpdecker) [inline test/run-css.js:60]: Why are you passing a require lookup?

File: test/run-css.js (hunk 1 above)
Comment (catesandrew) [inline test/run-css.js:35]: The default is '.'. That puts the fail.png images in the same directory as the screenshots. I put it there so when the resultant html is created, you can see the baseline, diff, and fail image all on the same row.

File: test/run-css.js (hunk 2 above)
Comment (catesandrew) [inline test/run-css.js:60]: Yeah that's a kluge. You should be able to require the files under phantom/src/modules, like fs, system, webpage, webserver and child_process, but for some unknown reason, the css-test.js file was not able to require it. It didn't make sense to me and I tried a few different things to no avail. In the end I used this kluge to pass it so we could use it when making the resultant output html page.

**Comment by kpdecker** [comment]
Lets hold off on this and do the original reporting. We can look into better reporting if we find that we can integrate this with QA well and we don't end up with tons of false positives to the extent of the tests not being helpful outside of large changes like the migration project.

**Pull request closed without merging**